presto-hive/src/main/java/com/facebook/presto/hive/HivePageSink.java

```diff
@@ -228,6 +238,19 @@ public long getValidationCpuNanos()
                 .mapToLong(HiveWriter::getValidationCpuNanos)
                 .sum();
 
+        if (waitForFileRenaming && verificationTasks.isEmpty()) {
+            ImmutableList.Builder<Slice> partitionUpdatesWithRenamedFileNames = ImmutableList.builder();
+            List<ListenableFuture<?>> futures = new ArrayList<>();
+            for (int i = 0; i < writers.size(); i++) {
```

Comment:
s/i/writerIndex

Reply:
Like i said above. If i use `i` as input param to lambda function then i am getting syntax error `Variable used in lambda expression should be final or effectivelyfinal`. Hence used `i` here and assigned it a new int variable `writerIndex` below.